core/arch/x86/decode.c

```diff
@@ -711,8 +823,11 @@ read_prefix_ext(const instr_info_t *info, decode_info_t *di)
     /* discard old info, get new one */
     int code = (int)info->code;
     int idx = (di->rep_prefix ? 1 : (di->data_prefix ? 2 : (di->repne_prefix ? 3 : 0)));
-    if (di->vex_encoded)
+    if (di->vex_encoded || di->evex_encoded) {
```

Comment:
So we're mixing vex and evex entries in the prefix_ext tables? The 'else' part is currently dead. I think the "|| di->evex_encoded" is a bug and you intended to add 4 entries to the prefix_ext tables for evex?

Reply:
Yes. I have added 4 entries to the prefix_ext tables for evex.